pkg/controller/kieapp/shared/utils.go

```diff
@@ -174,3 +180,51 @@ func GetNamespacedName(object metav1.Object) types.NamespacedName {
 		Namespace: object.GetNamespace(),
 	}
 }
+
+func GetConsoleYAMLSample(res resource.KubernetesResource) (*consolev1.ConsoleYAMLSample, error) {
+	snippetStr := res.GetAnnotations()["consoleSnippet"]
+	var snippet bool = false
+	if snippetStr != "" {
+		tmp, err := strconv.ParseBool(snippetStr)
+		if err != nil {
+			return nil, errors.New("Unable to parse snippet to boolean.")
+		}
+		snippet = tmp
+	}
+
+	title, _ := res.GetAnnotations()["consoleTitle"]
+	desc, _ := res.GetAnnotations()["consoleDesc"]
```

Comment:
I haven't seen how things should look in the web console... so I take it that title and description is optional? Fine if it is, just checking.

Reply:
They are optional.